src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -24,7 +24,8 @@ export class DocService {
   }
 
   urlGenerator(section: string, release = 'master'): string {
-    const domain = `http://docs.ceph.com/docs/${release}/`;
+    const docVersion = release === 'master' ? 'latest': release;
```

Comment:
```suggestion
    const docVersion = release === 'master' ? 'latest' : release;
```

This should fix the linting error.

Reply:
fixed and repushed.